lib/dal/src/workspace_snapshot/edge_weight.rs

```diff
@@ -93,7 +100,7 @@ impl EdgeWeightKind {
 
 #[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Eq)]
 pub struct EdgeWeight {
-    kind: EdgeWeightKind,
+    pub kind: EdgeWeightKind,
```

Comment:
Beginning to wonder why EdgeWeightKind isn't just EdgeWeight. but it's too late for that :)

Reply:
Yeah. Next graph migration, maybe :)